src/cljs_test_runner/main.clj

```diff
@@ -74,14 +77,17 @@
     :default "./test"]
    ["-o" "--out PATH" "The output directory for compiled test code"
     :default "./cljs-test-runner-out"]
+   ["-w" "--watch PATH" "Directory to watch for changes (alongside the src-path). May be repeated."
+    :assoc-fn (fn [m k v] (update-in m [k] (fnil conj [:src]) v))]
    ["-h" "--help"]])
 
 (defn -main
   "Creates a ClojureScript test runner and executes it with node."
   [& args]
-  (let [{:keys [options errors summary]} (cli/parse-opts args cli-options)]
+  (let [{:keys [options errors summary]} (cli/parse-opts args cli-options)
+        options                          (update options :watch (partial replace options))]
```

Comment:
alternative:
```clj
options (cond-> options (:watch options) (update :watch conj (:src options)))
```
This was quiet long but maybe less magical...

then `(fnil conj [:src])` would become `(fnil conj [])`